investment_tracking/serializers.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework import serializers
+from .models import InvestmentTracking
+
+class InvestmentTrackingSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = InvestmentTracking
+        fields = '__all__'
```

Comment:
Maybe it would be better to make two serializers: one for Create(SaveView) and one for Get(GetView)?